tools/webpack.config.js

```diff
@@ -329,19 +325,22 @@ const clientConfig = {
         try {
           const fileFilter = (file) => !file.endsWith(".map");
           const addPath = (file) => manifest.getPublicPath(file);
-          const chunkFiles = stats.compilation.chunkGroups.reduce((acc, c) => {
-            acc[c.name] = [
-              ...(acc[c.name] || []),
-              ...c.chunks.reduce(
-                (files, cc) => [
-                  ...files,
-                  ...Array.from(cc.files).filter(fileFilter).map(addPath),
-                ],
-                []
-              ),
-            ];
-            return acc;
-          }, Object.create(null));
+          const chunkFiles = stats.compilation.chunkGroups.reduce(
```

Comment:
Mmm, whale guts. This is some _syntax_!

Reply:
I don't think I wrote this code but thanks to Git blame, it's my fault now.